examples/plot_driver_laptimes.py

```diff
@@ -0,0 +1,54 @@
+"""Driver Laptimes Scatterplot
+==============================
+
+Plot a driver's lap times in a race.
+"""
+
+import fastf1 as f
+import fastf1.plotting as p
+import seaborn as sns
+from matplotlib import pyplot as plt
+
+p.setup_mpl(misc_mpl_mods=False)
+
+###############################################################################
+# Load the race session and create the plot
+
+race = f.get_session(2023, "Azerbaijan", 'R')
+race.load()
+fig, ax = plt.subplots(figsize=(8, 8))
+# sphinx_gallery_defer_figures
+
+###############################################################################
+# Get all the laps for a single driver
+# Filter out slow laps (yellow flag, VSC, pitstops etc.)
+# as they distort the graph axis
+
+driver_laps = race.laps.pick_driver("ALO").pick_quicklaps().reset_index()
+

```

Comment:
You need to add ``sphinx_gallery_defer_figures`` at the end of every block, else the previously deferred figure will be shown here.